### ISSUE TITLE
[Bug] [a11y] Consider making "Learn more" clickable item larger

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7716]
## Prerequisites
1. Make sure you have Google Accessibility Scanner installed.
2. Have a freshly installed build or a clean profile.

## Steps to reproduce
1. Launch Fenix. Go to private mode.
2. Type any letter in the search bar.
3. Using the Accessibility Scanner scan the page.

### Expected behavior
No suggestions are made.

### Actual behavior
The Accessibility Scanner suggests increasing the hight of the "Learn more" clickable item from 19dp to 48dp or larger - as seen in the screenshot below.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8), Samsung Galaxy Note 8 (Android 9)
* Fenix version: RC 3.0.2, Beta build 3.1.0-beta 3, Nightly build from 01/15

![Screenshot_20200115-152954_Accessibility Scanner](https://user-images.githubusercontent.com/40694158/72438833-5c43b600-37ae-11ea-8506-8c94ecc7f36c.jpg)

**Comment by sblatz** [comment]
Learn more should be a "button" or a "link"
Don't allow should be a "button"
Link should have 48 height

**Comment by mcarare** [comment]
Waiting on https://github.com/mozilla-mobile/fenix/pull/10288.

**Comment by sflorean** [comment]
@mcarare tested with Samsung Galaxy Note10 (Android 10) and now the item's height is 46dp.

**Comment by abodea** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #200728 from 7/28`** using the following devices:
• Google Pixel 3a (Android 10)
• Huawei Mate 20 Lite (Android 9)
• Google Pixel 4 XL (Android 11)

✔️ Checked it with the Accessibility Scanner and it falls in the 48dp threshold 

![screenshot_Firefox Nightly_2020-07-28-11_44_49](https://user-images.githubusercontent.com/42831109/88643218-1acf8400-d0ca-11ea-88ec-ecc71c3e9049.png)
